metadetect/lsst/masking.py

```diff
@@ -0,0 +1,48 @@
+EXPAND_RAD = 16  # confirm with Matt
+AP_RAD = 1
```

Comment:
how about expand_rad of 8 and ap_rad 1.5?

Reply:
There is also the scale of the deconvolution to consider. A pixel at the tail of the kernel in the deconvolved image will be effected by pixels that are another few kernel widths away. 

I think 8 pixels is too small because of this and would suggest going larger.

Reply:
I chose 16 and 1.5 for now